qcore/xyts.py

```diff
@@ -150,10 +322,20 @@ def corners(self, gmt_format=False):
         gmt_cnrs = "\n".join([" ".join(map(str, cnr)) for cnr in ll_cnrs])
         return ll_cnrs.tolist(), gmt_cnrs
 
-    def region(self, corners=None):
-        """
-        Returns simulation region as a tuple (x_min, x_max, y_min, y_max).
-        corners: use pre-calculated corners if given
+    def region(
+        self, corners: Optional[np.ndarray] = None
+    ) -> Tuple[float, float, float, float]:
+        """Returns simulation region.
+
+        Parameters
+        ----------
+        corners : Optional[np.ndarray]
+            If not None, use precalculated corners
```

Comment:
I think it might be helpful to make clearer that the precalculated corners are what can be passed in/given. 
Maybe something like: "If not None, use the _given_ precalculated corners"